sklearn/bench.py

```diff
@@ -71,16 +78,35 @@ def parse_args(parser, size=None, dtypes=None, loop_types=(),
                         help='Output CSV header')
     parser.add_argument('-v', '--verbose', default=False, action='store_true',
                         help='Output extra debug messages')
-
-    if dtypes is not None:
-        parser.add_argument('-d', '--dtype', default=np.dtype(dtypes[0]),
-                            type=np.dtype, choices=dtypes,
-                            help='Data type to use (see numpy.dtype docs)')
+    parser.add_argument('--data-format', type=str, default='numpy',
+                        choices=('numpy', 'pandas', 'cudf'),
+                        help='Data format: numpy, pandas, cudf')
+    parser.add_argument('--data-order', type=str, default='C',
+                        choices=('C', 'F'),
+                        help='Data order: F (column-major) or C (row-major)')
```

Comment:
It would be useful here to include what the default choice is, e.g. with adding something like `(default: %(default)s)` to the help text.

Reply:
Added